docs/can-guides/topics/debugging.md

```diff
@@ -331,3 +331,99 @@ Component.extend({
     }
 });
 ```
+
+## Installing CanJS Devtools
+
+There is a CanJS Devtools Chrome Extension available in the [Chrome Web Store](https://chrome.google.com/webstore/detail/canjs-devtools/hhdfadlgplkpapjfehnjhcebebgmibcb).
+
+Once installed, you will see a CanJS icon next to the address bar:
+
+<img src="../../docs/can-guides/images/devtools/canjs-button-disabled.png"
+  alt="The CanJS Devtools extension icon - disabled"
+  width="600px"/>
+
+Initially, this icon will be disabled. In order to enable the extension, you need to install `can-debug` as explained in [guides/debugging#SetupDebugging Setup Debugging]. Make sure you have `can-debug@1.2.0` or higher. Once this is set up, the icon will be enabled:
```

Comment:
You don’t need to change this, but in the future I would make links to a different section on the page just plain Markdown links, e.g. `[Setup Debugging](#SetupDebugging)`